test/Grains/TestGrainInterfaces/ProgramaticStreamSubscribe/IPassive_ConsumerGrain.cs

```diff
@@ -26,7 +26,7 @@ public interface IJerk_ConsumerGrain : IGrainWithGuidKey, IStreamSubscriptionObs
     {
     }
 
-    public interface IImplicitSubscribeGrain: IGrainWithGuidKey
+    public interface IImplicitSubscribeGrain: IPassive_ConsumerGrain
```

Comment:
I don't think IPassive_ConsumerGrain should (nor should any grain interface) inherit from IStreamSubscriptionObserver.  Implementations should inherit from it, but not grain interfaces.